cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReference.groovy

```diff
@@ -0,0 +1,28 @@
+package org.jenkinsci.plugins.workflow.cps.Snippetizer
+
+
+def st = namespace("jelly:stapler")
+
+p {
+    a(href: "${rootUrl}${my.DSL_HELP_URL}", target: "_blank") {
```

Comment:
And BTW from Groovy you can use real Java syntax. So `my.DSL_HELP_URL` could be replaced with `Snippetizer.DSL_HELP_URL` probably.